components/PrintableForm/PrintableForm.tsx

```diff
@@ -0,0 +1,59 @@
+import s from './PrintableForm.module.scss';
+import { Form } from 'data/flexibleForms/forms.types';
+
+interface Props {
+  form: Form;
+}
+
+const PrintableForm = ({ form }: Props): React.ReactElement => (
```

Comment:
We need to include information about the resident in here somewhere

Reply:
the need isn't for specific forms as far as i can tell, just to print them off and use them as they would paper forms - we're probably okay with just blank fields

Reply:
Ok, as a preliminary version that can be modified to later include resident information seems fine